api/src/certification/enrolment/infrastructure/repositories/candidate-repository.js

```diff
@@ -20,7 +21,65 @@ const findBySessionId = async function ({ sessionId }) {
   return results.map(_toDomain);
 };
 
-export { findBySessionId };
+/**
+ * @function
+ * @param {Object} params
+ * @param {number} params.certificationCandidateId
+ *
+ * @return {Candidate}
+ */
+const get = async function ({ certificationCandidateId }) {
+  const certificationCandidate = await knex('certification-candidates')
+    .where({
+      id: certificationCandidateId,
+    })
+    .first();
+
+  return _toDomain(certificationCandidate);
+};
+
+/**
+ * @function
+ * @param {Object} certificationCandidate
+ *
+ * @return {Candidate}
+ * @throws {CertificationCandidateNotFoundError} Certification candidate not found
+ */
+const update = async function (certificationCandidate) {
+  const [updatedCertificationCandidate] = await knex('certification-candidates')
+    .where({
+      id: certificationCandidate.id,
+    })
+    .update({ ...certificationCandidate })
```

Comment:
Proposition : dans le cas d'un model non anémique, éviter le mapping via spread operator au profit d'un mapping explicite.

```suggestion
    .update({ name : certificationCandidate.name, xxx : certificationCandidate.yyy })
```

J'y vois un apport de :
* moins de mental mapping
* de couplage domain/infra
* et la réduction d'effet de bord liés au langage (parce exemple les getter ne sont pas lus par le spread operator)

Reply:
@Steph0 Au final vu qu'on veut avoir un update générique et non spécifique, on va devoir énumérer toutes les propriétés du model. ça n'est pas plus simple d'utilier le `spread` pour le même résultat ?